README.md

```diff
@@ -251,6 +253,86 @@ capability might be supported but not enabled.
 | JSCVT     | Perform Conversion to Match Javascript
 | DCPOP     | Persistent Memory Support
 
+### Custom Features
+The Custom feature source allows the user to define features based on a mix of predefined rules.
+
+#### Rule Types
+
+##### PciId Rule
+The PciId rule allows matching the PCI devices in the system on the following PCI attributes: `class`,`vendor` and
+`device` identifiers.
+
+```yaml
+pciId :
+  class: ["<PCI_CLASS_ID>", ...]
+  vendor: ["<PCI_VENDOR_ID>", ...]
+  device: ["<PCI_DEVICE_ID>", ...]
+```
+Matching is done by performing a logical OR for each attribute and logical AND between the specified attributes for
+each PCI device in the system.
+
+##### LoadedKMod Rule
+The LoadedKMod rule allows matching the loaded kernel modules in the system against a provided list of kernel modules.
+
+```yaml
+loadedKMod : ["<KERNEL_MODULE>", ...]
```

Comment:
I'd re-format this, too, at least drop the quotes

Corresponding the suggestion for PciId rule above:
```yaml
loadedKMod : [<module name>, ...]
```